evaluators/rank/f1ScoreEvaluator/__init__.py

```diff
@@ -0,0 +1,39 @@
+from jina.executors.evaluators.rank import BaseRankingEvaluator
+
+class f1ScoreEvaluator(BaseRankingEvaluator):
+    """
+    :class:`f1ScoreEvaluator` Gives the f1 score of a result given the groundtruth..
+    """
+
+    def __init__(self, *args, **kwargs):
```

Comment:
for our tests to pass, we need to provide a valid `eval_at` default, we can pass it in the `config.yml` or directly in the `__init__` of our class which we are doing right now in other PRs.